### ISSUE TITLE
More entities and training

### ISSUE THREAD

**Issue by arjunmenon** [explosion/displacy-ent#1]
The [demo page](https://demos.explosion.ai/displacy-ent/?text=buy%2012%20dollars%20worth%20of%20Apple%20stock%20tomorrow%20night&ents=person%2Cnorp%2Cfacility%2Corg%2Cgpe%2Cloc%2Cproduct%2Cevent%2Cwork_of_art%2Clanguage%2Cdate%2Ctime%2Cpercent%2Cmoney%2Cquantity%2Cordinal%2Ccardinal&model=en) shows many different types of entities. Can they be available from the library as well.

Is it possible to train the library on custom datasets? This would make it doubly cool.

**Comment by ines** [comment]
Yes, the entity types used in the demo are the built-in entity types of spaCy. You can read more about this in the [entity recognition usage workflow](https://spacy.io/docs/usage/entity-recognition). It also explains the training process. Once you're done with training, you can run our [displaCy REST service](https://github.com/explosion/spacy-services) and hook it up with the displaCy ENT (using the `/ent/` endpoint).

You should also check out @tcrossland's [trainable displaCy ENT server](https://github.com/explosion/spacy-dev-resources/tree/master/training/displacy_ent) – it even comes with a cool [annotation front-end](https://github.com/tcrossland/spacy-annotator) that uses this visualiser.

**Comment by tcrossland** [comment]
I haven't actually integrated the front end with the server yet, but it should be fairly straightforward. Will get round to it next week probably.

**Comment by nishankmahore** [comment]
Follow This link
https://spacy.io/docs/usage/entity-recognition#updating
